dist/dattobd.spec

```diff
@@ -205,6 +213,8 @@ Group:           System Environment/Kernel
 %endif
 %endif
 
+BuildRequires:   rsync
```

Comment:
Please put this back where it was. This change is superfluous.

Reply:
Is it? It won't build where it is on master:
```
This is debbuild, Version 17.5.0
dpkg-query: no packages found matching rsync
 * Missing build-dependency rsync!
To install all missing dependencies, run 'apt install rsync'.
Can't build dattobd:  build requirements not met.
make: *** [Makefile:56: deb] Error 1
```

Reply:
That's because you have `dpkg-query` installed and it can't find the dependency, and you're not on a Debian system. There's not a `--nodeps` switch to make that work properly on a non-Debian system.

Reply:
It's basically expected behavior.